test/support/io-helpers.js

```diff
@@ -1,10 +1,14 @@
-import {times} from "lodash";
+const times = require('lodash/times');
```

Comment:
Why?

Reply:
Not at my laptop so can't confirm this, but I made it a rule that only `src/` gets processed through Babel, so files outside of it have to use the syntax supported by Node 6.14. I can't figure out exactly if `import` is supported, but my guess is it wasn't and I had to switch. But I could be wrong. I can revert it and see if imports work properly when I get in in a couple hours.

Reply:
(Turns out imports are supported so I fixed this.)